datum/dsl/org/openstax/ec/nudged/v1/nudged.rb

```diff
@@ -0,0 +1,13 @@
+namespace 'org.openstax.ec.nudged.v1'
+
+fixed :uuid, 16
```

Comment:
Will we want to replace this with `require 'avro_types'` so we don't repeat this definition all over?